docs/model/product-filtering.md

```diff
@@ -0,0 +1,44 @@
+# Product Filtering
```

Comment:
You should change the name of the article to Front-end Product Filtering.

Reply:
renamed to Front-end Product Filtering (including file name and links to this file)